lib/components/src/blocks/IFrame.tsx

```diff
@@ -27,7 +27,10 @@ export class IFrame extends Component<IFrameProps> {
   }
 
   shouldComponentUpdate(nextProps: IFrameProps) {
-    const { scale } = nextProps;
+    const { scale, src } = nextProps;
```

Comment:
@ndelangen can you look at this?

Reply:
The way to reproduce this is described in #10851
that granular MDX docpage shows how the Story angular-iframe is not refreshed

Reply:
I think it'd be better to set a key on the iframe based on the storyId.

If the key changes, react will unmount & create a new element.

I think this totally defeats the purpose of the `shouldComponentUpdate`

Reply:
@ndelangen the StoryId is inside `nextProps.id`.
I'm not a React developer so I'm not familiar at all with the lifecycle.